app/src/main/java/com/eveningoutpost/dexdrip/stats/StatsResult.java

```diff
@@ -99,11 +109,11 @@ public int getIn() {
         return in;
     }
 
-    public int getTotal_carbs() {
+    public double getTotal_carbs() {
         if (total_carbs < 0) {
             Cursor cursor = Cache.openDatabase().rawQuery("select sum(carbs) from treatments  where timestamp >= " + from + " AND timestamp <= " + to, null);
             cursor.moveToFirst();
-            total_carbs = cursor.getInt(0);
+            total_carbs = cursor.getDouble(0);
```

Comment:
This is now a double but rounded later when displayed? Can you explain the difference here, I'm not sure if I'm missing something.